internal/pithosctl/pkg/pithos/pithos.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pithos
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"os"
+	"text/template"
+	"time"
+
+	"github.com/gravitational/pithos-app/internal/pithosctl/pkg/config"
+	"github.com/gravitational/pithos-app/internal/pithosctl/pkg/kubernetes"
+
+	"github.com/gravitational/rigging"
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+const (
+	templateFile      = "/var/lib/gravity/resources/pithos-cfg/config.yaml.template"
+	initJobFile       = "/var/lib/gravity/resources/pithos-initialize.yaml"
+	masterTenantName  = "ops@gravitational.io"
+	regularTenantName = "pithos"
+	configMapName     = "pithos-cfg"
+	retryAttempts     = 60
+	retryPeriod       = 5 * time.Second
+)
+
+// Control defines configuration for operations
+type Control struct {
+	cfg    config.Pithos
+	client *k8s.Clientset
+}
+
+// NewControl creates new Control object
```

Comment:
Not a very helpful doc-comment. Can you add what the new instance of `Control` is useful for (like what does it control)?